src/uwtools/tests/drivers/test_jedi.py

```diff
@@ -176,7 +177,8 @@ def file(path: Path):
             cmds = [
                 "module load some-module",
                 "module load jedi-module",
-                "time /path/to/qg_forecast.x --validate-only %s 2>&1" % (cfgfile),
+                "time %s --validate-only %s 2>&1"
+                % (driverobj._driver_config["execution"]["executable"], cfgfile),
```

Comment:
We can get the executable name from the driver config and avoid repeating it here.